src/views/buttons.tsx

```diff
@@ -252,7 +252,7 @@ const Buttons: React.FunctionComponent<ButtonsComponentProps> = () => {
                   borderColor: 'rgba(78, 116, 289, 1)',
                 }}
                 type="outline"
-                raised="true"
```

Comment:
Shouldn't it be `raised={true}` ?

Reply:
It works either way actually. When you pass a boolean parameter without a value it is implied to be attached a true value. You can read more about it here https://dzone.com/articles/react-guide-to-props-part-i

Reply:
Yes, I know it works either way (I did check both live on Expo). I said it so that it is uniform with the existing code (see line 221 for `disabled`)

Reply:
Ahh okay. Sure, I can change it to what you suggested! Thanks!

Reply:
Done!